### PR TITLE
Populate release notes from changelog

### LINKED ISSUE #1030: Populate GitHub Release Notes from Changelog
Our [release workflow](https://github.com/phylum-dev/cli/blob/fd6e2e2d8f8288c2c3baa2218f4238ca8e49a034/.github/workflows/release.yml#L141-L155) creates the release and currently uses `generate_release_notes` to fill in the body of the release.

We now have a much more detailed changelog since #1024 was merged. So we should populate our release notes from the changelog instead.

Ref: [Create a release](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) in GitHub's REST API docs

### PR DESCRIPTION
**Pull request by kylewillmon**
Closes #1030

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -138,18 +138,39 @@ jobs:
...
       - name: Create GitHub release
         id: create_release
         uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
         with:
           # The response is explicitly returned here so it will be available for other steps
           script: |
+            const fs = require("node:fs");
+
+            const tag = process.env.GITHUB_REF_NAME;
+            const prerelease = tag.includes("-rc");
```
Comment (maxrake) [inline .github/workflows/release.yml:155]:
```suggestion
            const prerelease = tag.includes("rc");
```
The other checks for a pre-release in this file do not include the hyphen/dash/whatever.

**Review by maxrake — DISMISSED** [review]
It looks like it will work...but we'll know for sure at the next release! Only one minor suggestion...take it or leave it.

LGTM.

**Comment by kylewillmon** [comment]
I changed my approach in order to simplify what is added to the `Release` job (since that rarely gets run). I'm now adding the release notes as an artifact in the earlier `Build-Release-Artifacts` job. I've kicked off a test by pushing a branch named `v5.0.0` (**not a tag** because that tag already exists) and dispatching the release workflow on that branch. You can see that test [here](https://github.com/phylum-dev/cli/actions/runs/4735983617)

**Review by maxrake — APPROVED** [review]
The test run was reviewed...it worked as expected.

LGTM.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -114,6 +114,37 @@ jobs:
...
           retention-days: 7
 
+      - name: Create release notes
+        uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
+        with:
+          script: |
+            const fs = require("node:fs");
+
+            const tag = process.env.GITHUB_REF_NAME;
+            const prerelease = !tag.match(/^v(\d+\.)*\d+$/);
```
[the remark this answers is not in the document]
Comment (maxrake) [reply .github/workflows/release.yml:124]: Good idea.

**Pull request merged**